ArduCopter/motors.pde

```diff
@@ -152,30 +158,6 @@ static bool init_arm_motors(bool arming_from_gcs)
     }
     calc_distance_and_bearing();
 
-    uint32_t gyro_cal_time = 0;
-    if(did_ground_start == false) {
-        uint32_t gyro_cal_begin = millis();
-
-        startup_ground(true);
```

Comment:
I'll check the code, but I thought they already set to the correct values on intialisation. I have done several flights using this code and no issues with ground effect compensation or land detector observed so far.